lib/js/jumpstart_ui.behavior.js

```diff
@@ -10,12 +10,16 @@
 export default {
 
   // Attach Drupal Behavior.
-  attach (context, settings) {
-    // console.log("Attached.");
+  attach(context, settings) {
+    (function ($) {
+      $('figure .media-entity-wrapper.video', context).each((i, video) => {
+        $(video).closest('figure').css('width', '100%');
+      })
```

Comment:
Do you need a ; here?

Reply:
Don't really _need_ one. Yay for javascript being the most unstructured code. :rofl:

Reply:
Well. True. But we usually do have one. :D

Reply:
You got it :+1:

Reply:
Thank you!